examples/pedersen_commitment.rs

```diff
@@ -22,19 +22,18 @@ where
 
     let security_bits = 256;
     let blinding_factor = BigInt::sample(security_bits);
-    let com = PedersenCommitment::<P>::create_commitment_with_user_defined_randomness(
+    let _ = PedersenCommitment::<P>::create_commitment_with_user_defined_randomness(
         message,
         &blinding_factor,
     );
-    (com, blinding_factor);
```

Comment:
I think here the error was that we don't use the result, but it shouldn't be discarded. It would be better to print the resulted commitment and blinding factor

Reply:
Done. With this applied we get:

```
cargo run --example pedersen_commitment secp256k1
   Compiling curv v0.5.6 (/home/tobin/build/github.com/tobin-crypto/curv)
    Finished dev [unoptimized + debuginfo] target(s) in 0.91s
     Running `/home/tobin/.cache/cargo/debug/examples/pedersen_commitment secp256k1`

created commitment with user defined randomness 

 blinding_factor 106370464485331616747936628116362783293199927728552668243993612578919180718169 
 commitment: Secp256k1Point {
    purpose: "combine",
    ge: PublicKey(
        b25411d0fc67983752cc11b5fddd3c9ea0617429bd35578a377aad5c59a0011de115cec6daeaee75a58d8013881a099d3d84d0760e775f581353e3d78b6431e2,
    ),
}
```